designdocs/capacity_staking_rewards_implementation.md

```diff
@@ -0,0 +1,274 @@
+# Capacity Staking Rewards Implementation
```

Comment:
I'm not sure why this makes it look like the entire design doc is new.  There were a lot of changes but not to every line.